README.md

```diff
@@ -4,8 +4,8 @@ platforms: dotnet
 author: jmprieur
 level: 200
 client: Mobile (UWP)
-service: ASP.NET Web API
-endpoint: AAD V1
+service: ASP .NET Web API

```

Comment:
Actually, @kjyam98 ASP.NET does not have a space between ASP and .NET (this is a trade mark)